src/components/Pelaporan/FormCloseContact.vue

```diff
@@ -0,0 +1,199 @@
+                    <v-col cols="12" md="3" sm="12" :class="{'py-0': $vuetify.breakpoint. smAndDown}">
+                      <label>{{ $t('label.activities_carried_out') }}</label>
+                    </v-col>
+                    <v-col cols="12" md="9" sm="12" :class="{'py-0 pb-3': $vuetify.breakpoint. smAndDown}">
+                      <ValidationProvider>
+                        <v-text-field v-model="data.activity" solo-inverted />
+                      </ValidationProvider>
+                    </v-col>
+                  </v-row>
+                  <v-row align="start">
+                    <v-col cols="12" md="3" sm="12" :class="{'py-0': $vuetify.breakpoint. smAndDown}">
+                      <label>{{ $t('label.contact_date') }}</label>
+                    </v-col>
+                    <v-col cols="12" md="9" sm="12" :class="{'py-0 pb-3': $vuetify.breakpoint. smAndDown}">
+                      <input-date-picker :format-date="formatDate" :label="$t('label.contact_date')" :date-value="data.contact_date" :value-date.sync="data.contact_date" @changeDate="handeChangeContactDate($event, index)" />
+                    </v-col>
+                  </v-row>
+                </v-form>
+              </v-container>
+            </v-card>
+          </v-container>
+        </div>
+        <v-container fluid>
+          <v-card outlined class="text-center" color="#F5F5F5">
+            <v-btn class="ma-2 mt-5" outlined small fab @click="handleAddFormCloseContact">
+              <v-icon>mdi-plus</v-icon>
+            </v-btn>
+            <p>Tambah Data Kontak</p>
+          </v-card>
+        </v-container>
+      </v-card>
+    </ValidationObserver>
+  </v-container>
+</template>
+
+<script>
+import { ValidationObserver, ValidationProvider } from 'vee-validate'
+export default {
+  name: 'FormCloseContact',
+  components: {
+    ValidationObserver,
+    ValidationProvider
+  },
+  props: {
+    formPasien: {
+      type: Object,
+      default: null
+    }
+  },
+  data() {
+    return {
+      isAddCloseContactForm: false,
+      totalCloseContact: 0,
+      idCloseContactForm: 0,
+      isValid: false,
+      showAlert: false,
+      formatDate: 'YYYY/MM/DD'
+    }
+  },
+  mounted() {
+    this.handleAddFormCloseContact()
+  },
+  methods: {
+    handleAddFormCloseContact() {
+      this.isValid = true
+      this.showAlert = false
+      this.idCloseContactForm = this.idCloseContactForm + 1
+      this.formPasien.close_contact_patient.push({
+        id: this.idCloseContactForm,
+        name: '',
+        phone_number: '',
+        gender: '',
+        years_old: '',
+        months_old: '',
+        address: '',
+        related: '',
+        activity: '',
+        contact_date: ''
+      })
+    },
+    handleDeleteFormCloseContact(index) {
+      this.formPasien.close_contact_patient.splice(index, 1)
+      if (this.formPasien.close_contact_patient.length === 0) {
+        this.isValid = false
+      }
```

Comment:
ini mungkin bisa dipersingkat jadi gini kak.. bisa ga ya kalo di vue?

```suggestion
      this.isValid = this.formPasien.close_contact_patient.length !== 0
```

Reply:
hrsnya bisa sih, nnti saya coba dlu

Reply:
itu shorthand auto `true` atau `false`